at-tanvo/D3-Exercise5.rb

```diff
@@ -0,0 +1,10 @@
+puts "Nhap phan tu mang :"
+a = gets.to_i
+arr = []
+for i in 1..a
+  b = gets.to_i
+  arr.push(b)  
+end
+hash_count = Hash.new
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/7587b87b7304d151b3838535fcf67736f443399f/at-tanvo/D3-Exercise5.rb#L8"></span>Style/EmptyLiteral: Use hash literal <code>{}</code> instead of <code>Hash.new</code>. (https://github.com/rubocop-hq/ruby-style-guide#literal-array-hash)</td>
    </tr>
  </tbody>
</table>